samples/star-wars-api/HttpHandlers.fs

```diff
@@ -60,9 +60,20 @@ module HttpHandlers =
 
             let removeWhitespacesAndLineBreaks (str : string) = str.Trim().Replace ("\r\n", " ")
 
+            let hasData (context: HttpContext) =
+                match context.Request.Body with
+                | :? System.IO.Stream as stream ->
+                    if stream.CanSeek then
+                        stream.Length > 0L
+                    else
+                        let buffer = Array.zeroCreate 1
+                        let valu = stream.Read(buffer, 0, 1)
```

Comment:
But Gitaffe binds model from the original stream.
Maybe you need to seek to 0 and replace NetworkStream with your MemoryStream in the HTTP response?

Reply:
Ah, interesting idea. So read only a single byte here.maybe it is ok